package.json

```diff
@@ -52,7 +52,7 @@
     "minify:css:cdn": "cleancss dist/alt/video-js-cdn.css -o dist/alt/video-js-cdn.min.css",
     "minify:css:default": "cleancss dist/video-js.css -o dist/video-js.min.css",
     "watch": "npm-run-all -p watch:*",
-    "watch:lang": "chokidar --initial 'lang/**/!(zh-Hans|zh-Hant)*.json' -c 'npm run build:lang'",
+    "watch:lang": "chokidar --initial \"lang/**/!(zh-Hans|zh-Hant)*.json\" -c \"npm run build:lang\"",
```

Comment:
nitpick: Can this change be reversed? Seems less readable this way.

Reply:
thanks. Actually that change was needed for us in order to pass the build on Windows. It was failing because of that. I do realize it seems less readable.  
Just let us know if you want to revert it nonetheless.

Reply:
Nope, that's okay. Thanks!